python-avd/pyavd/_anta/index.py

```diff
@@ -29,6 +29,11 @@
         conditional_keys=[StructuredConfigKey.ROUTER_AVT],
         input_factory=VerifyAVTRoleInputFactory,
     ),
+    TestSpec(
+        test_class=VerifyAVTSpecificPath,
+        conditional_keys=[StructuredConfigKey.ROUTER_AVT_VRFS, StructuredConfigKey.ROUTER_PATH_SELECTION_PATH_GROUPS],
```

Comment:
Are there scenarios where we can have `router_adaptive_virtual_topology` without any `vrfs`? What about `router_path_selection`, can we have it without any `path_groups` configured? If not, we should probably just reuse the same conditional keys and let the factory logs what's missing:
```suggestion
        conditional_keys=[StructuredConfigKey.ROUTER_AVT, StructuredConfigKey.ROUTER_PATH_SELECTION],
```

Reply:
yes not via the eos_designs inputs, but can with `custom_structured_configuration`. I added the `custom_structured_configuration_router_adaptive_virtual_topology` in dc1-leaf1a device for a test where static_peers should not rendered in structured_config so test can skip.